src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-realm-form/rgw-multisite-realm-form.component.html

```diff
@@ -1,58 +1,77 @@
+  <form #frm="ngForm"
+        #formDir="ngForm"
+        [formGroup]="multisiteRealmForm"
+        novalidate>
+  <div cdsModalContent>
+    <div cdsCol
+    [columnNumbers]="{md: 4}">
+              <div class="form-item">
+                <cds-text-label for="realmName"
+                                cdRequiredField="Realm Name"
+                                [invalid]="!multisiteRealmForm.controls.realmName.valid && multisiteRealmForm.controls.realmName.dirty"
+                                [invalidText]="realmNameError"
+                                i18n>Username
+                  <input cdsText
+                         placeholder="Realm name..."
+                         i18n-placeholder
+                         id="realmName"
+                         name="realmName"
+                         formControlName="realmName"
+                         [invalid]="!multisiteRealmForm.controls.realmName.valid && multisiteRealmForm.controls.realmName.dirty"
+                         autofocus>
+                </cds-text-label>
+                <ng-template #realmNameError>
+                  <span *ngIf="multisiteRealmForm.showError('realmName', formDir, 'required')">
+                    <ng-container i18n>
+                     This field is required.
+                    </ng-container>
+                  </span>
+                  <span *ngIf="multisiteRealmForm.showError('realmName', formDir, 'uniqueName')">
+                    <ng-container i18n>
+                      The chosen realm name is already in use.
+                    </ng-container>
+                  </span>
+                </ng-template>
+              </div>
+              <div class="bx--form-item inline-helper">
+        <cds-checkbox 
+                      id="default_realm"
                    name="default_realm"
                    formControlName="default_realm"
-                   [attr.disabled]="action === 'edit' ? true: null"
-                   type="checkbox">
-            <label class="form-check-label"
-                   for="default_realm"
-                   i18n>Default</label>
-            <cd-helper *ngIf="action === 'edit' && info.data.is_default">
-              <span i18n>You cannot unset the default flag.</span>
-            </cd-helper>
-            <cd-helper *ngIf="action === 'edit' && !info.data.is_default">
-              <span i18n>Please consult the&nbsp;<a href="{{ docUrl }}">documentation</a>&nbsp;to follow the failover mechanism</span>
-            </cd-helper>
-            <cd-helper *ngIf="defaultRealmDisabled && action === 'create'">
-              <span i18n>Default realm already exists.</span>
-            </cd-helper>
-          </div>
-        </div>
+                   [attr.disabled]="realmData.action === 'edit' ? true: null"
+                      i18n>Default
+        </cds-checkbox>
+        <cd-helper *ngIf="realmData.action === 'edit' && realmData.info.data.is_default" class="helper-text">
```

Comment:
let's remove the `cd-helper` here and use the `cd-help-text`  to show the helper text outright. 

you can check here for an example: https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html#L308

Reply:
@nizamial09, seems like **cd-help-text** is not working like **cd-helper.** .In case of cd-help-text just text is getting printed.I have taken reference from other components eg- Rgw-bucket-form. Can you please help me with the same.

![Bucket](https://github.com/user-attachments/assets/9212b727-43a7-4fee-bba8-706fec66d8a3)

Reply:
the intention of the `cd-help-text` is just that. To print the text as it is rather than hiding it inside an info icon. That's what you are seeing in the bucket-form as well isn't it? 

we have two things here:
1. cd-helper: which shows a info icon and let's you hover over it and show the information
2. cd-help-text: directly show the help text in the form.

carbon follows the second one when the information is crucial and needs to show outright. so let's apply that here.

Reply:
if i am missing something, can you put the screenshots so I can understand it better, thanks.

Reply:
Ok,got it. I was trying to match existing UI which shows info icon.